airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -344,6 +402,10 @@ def _build_cluster_data(self):
             "autoscaling_config": {},
             "endpoint_config": {},
         }
+
+        if self.min_num_workers:
```

Comment:
Should we raise an Exception at initialization time if `min_num_workers > num_workers`?

It's usually best to surface issues as early as possible.

Reply:
Done